.eleventy.js

```diff
@@ -2,6 +2,17 @@ import govukEleventyPlugin from "@x-govuk/govuk-eleventy-plugin";
 
 export default function (eleventyConfig) {
 
+    // The name of the repo as it appears on GitHub
+    const repoName = '';
+
+    const url = process.env.GITHUB_ACTIONS
+        ? `https://ukhomeoffice.github.io/${repoName}/`
+        : '/';
+
+    const pathPrefix = process.env.GITHUB_ACTIONS
+        ? `/${repoName}/`
+        : '/';
```

Comment:
```suggestion
    /** TODO: If the site is not hosted in the root of the host domain, this should be 
      * the path to the root of the site.
      * 
      * For example when using a GitHub action to deploy to GitHub pages:
      * 
      * ```javascript
      * const pathPrefix = process.env.GITHUB_ACTIONS
      *    ? `/${repoName}/`
      *    : '/';
      */
    const pathPrefix = '/';
```